templates/article/article.tpl

```diff
@@ -6,15 +6,25 @@
  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
  *
  * Article View.
+ *
+ * Available data:
+ *  $article Article The article object for the current article view
+ *  $galley ArticleGalley The (optional!) galley object for the current view
+ *  $galleys array The list of galleys available for this article
+ *  $citationsFactory ItemIterator List of citations from this article.
+ *  $subscriptionRequired boolean Whether or not a subscription is required
+ *  $subscribedUser boolean Whether or not the current user is subscribed
+ *  $subscribedDomain boolean Whether or not the user has subscription access by domain
+ *  $showGalleyLinks boolean True iff OJS should display galley links regardless of subscription status
  *}
-{if $galley}
-	{assign var=pubObject value=$galley}
-{else}
-	{assign var=pubObject value=$article}
-{/if}
-
-
-{include file="article/header.tpl"}
+{strip}
+	{if $galley}
+		{assign var=pubObject value=$galley}
+	{else}
+		{assign var=pubObject value=$article}
+	{/if}
+	{include file="article/header.tpl"}
+{/strip}
```

Comment:
This is just a minor bit of cleanup. We use {strip} here and there to prevent whitespace from getting in ahead of the header template, but I've noticed a few spots where that's not necessary and I've been removing it in cases like that. (Whitespace ahead of the <!DOCTYPE> sometimes causes problems IIRC.)